sdk/src/Core/Amazon.Runtime/Internal/Util/ILogger.cs

```diff
@@ -23,6 +23,5 @@ public interface ILogger
         void Debug(Exception exception, string messageFormat, params object[] args);
         void DebugFormat(string messageFormat, params object[] args);
         void Error(Exception exception, string messageFormat, params object[] args);
-        void Flush();
```

Comment:
Flush is not supported in Microsoft.Extensions.Logger and we only used in 2 places in the entire SDK that I don't think needed to be used there.